swarm.sh

```diff
@@ -34,16 +34,22 @@ main() {
     if [[ "${ACTION}" == "up" ]]; then
       clean_containers_and_configs
     fi
+
     log info "Setting config digests"
     config::set_config_digests "$COMPOSE_FILE_PATH"/importer/docker-compose.config.yml
-    try "docker stack deploy -c ${COMPOSE_FILE_PATH}/importer/docker-compose.config.yml instant" "Failed to deploy Cares on Platform"
+    # Clickhouse 
+    config::generate_service_configs cares-clickhouse-config-importer / "${COMPOSE_FILE_PATH}/importer/analytics-datastore-clickhouse/" "${COMPOSE_FILE_PATH}/importer"
+    ClickhouseTempComposeParam="-c ${COMPOSE_FILE_PATH}/importer/docker-compose.tmp.yml"
+
+    try "docker stack deploy -c ${COMPOSE_FILE_PATH}/importer/docker-compose.config.yml $ClickhouseTempComposeParam instant" "Failed to deploy Cares on Platform"
```

Comment:
Let's try be consistent with our naming conventions... `ClickhouseTempComposeParam` > `clickhouse_temp_compose_param`

Reply:
done :)